### PR TITLE
Add tests for BaseService + fix hex colors

### PR DESCRIPTION
**Pull request by paulmelnikow**
Breaking `invokeHandler` into its own function makes it easy to test the error handling, and also to expand the error handling as I've done in this WIP branch: https://github.com/badges/shields/compare/master...paulmelnikow:rewrite-npm-typedefs

### CODE REVIEW

**Comment by paulmelnikow** [comment]
After this lands I'll open a PR to handle query parameters in BaseService. It'll conflict if I do it now.

File: services/base.js [hunk 1]
```diff
@@ -87,6 +87,15 @@ module.exports = class BaseService {
     return result;
   }
 
+  async invokeHandler(namedParams) {
+    try {
+      return await this.handle(namedParams);
+    } catch (error) {
+      console.log(error);
+      return { message: 'error' };
```
Comment (chris48s) [inline services/base.js]: Lets not scope creep this PR. I think this is fine for refactoring + adding tests but an improvement we should make here soon is the ability to customise this error based on the exception.

**Comment by paulmelnikow** [comment]
Thanks for reviewing!
